paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -24,11 +24,21 @@ import app.cash.paparazzi.accessibility.RenderSettings.DEFAULT_RECT_SIZE
 import app.cash.paparazzi.accessibility.RenderSettings.DEFAULT_TEXT_COLOR
 import app.cash.paparazzi.accessibility.RenderSettings.DEFAULT_TEXT_SIZE
 import app.cash.paparazzi.accessibility.RenderSettings.getColor
+import java.awt.Font
 import java.awt.image.BufferedImage
 
 class AccessibilityRenderExtension : RenderExtension {
   private val accessibilityTree = LinkedHashSet<View>()
 
+  // We have seen issues with the built in font rendering incorrectly on different OS (Linux, Mac OS, etc.).
+  // We will provide a roboto font that is monospaced.
+  private val robotoFont: Font by lazy {
+    val inputStream = javaClass.getResourceAsStream("/font/RobotoMono-Medium.ttf")
```

Comment:
paparazzi already packages the android platform fonts.  can we use that instead of re-bundling?

Reply:
Ya good call, updated to use one of the bundled fonts.